unofficial/c511027118.lua

```diff
@@ -0,0 +1,66 @@
+--オーバーレイ・ウェッジ
+--Overlay Wedge
+--Scripted by The Razgriz
+local s,id=GetID()
+function s.initial_effect(c)
+	Duel.EnableGlobalFlag(GLOBALFLAG_DETACH_EVENT)
+	--Activate
+	local e1=Effect.CreateEffect(c)
+	e1:SetCategory(CATEGORY_DISABLE)
+	e1:SetType(EFFECT_TYPE_ACTIVATE)
+	e1:SetCode(EVENT_CHAINING)
+	e1:SetCondition(s.condition)
+	e1:SetTarget(s.target)
+	e1:SetOperation(s.activate)
+	c:RegisterEffect(e1)
+	if not s.global_check then
+		s.global_check=true
+		c511027118[0]=nil
+		local ge1=Effect.CreateEffect(c)
+		ge1:SetType(EFFECT_TYPE_FIELD+EFFECT_TYPE_CONTINUOUS)
+		ge1:SetCode(EVENT_DETACH_MATERIAL)
+		ge1:SetOperation(s.checkop)
+		Duel.RegisterEffect(ge1,0)
+	end  
+end
+function s.checkop(e,tp,eg,ep,ev,re,r,rp)
+	local cid=Duel.GetCurrentChain()
+	if cid>0 then
+		c511027118[0]=Duel.GetChainInfo(cid,CHAININFO_CHAIN_ID)
+	end
+end
+function s.condition(e,tp,eg,ep,ev,re,r,rp)
+	return Duel.GetChainInfo(0,CHAININFO_CHAIN_ID)==c511027118[0] and Duel.IsChainDisablable(ev) 

```

Comment:
Having
```lua
re:IsActiveType(TYPE_XYZ)
```
would be better incase Xyz Monster lost Xyz Type (looking at you , En Birds)